### PR TITLE
Support pluginGroups

### PR DESCRIPTION
**Pull request by uportalis**
As a user of this GitHub Action, I need to be able to generate Maven settings so that I can use my custom Maven plugins.

In order to fulfill this need, this Pull Request adds the support for the `pluginGroups` element in the Maven settings.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -45,6 +45,11 @@ Reference: [Maven Settings > Plugin Repositories](http://maven.apache.org/settin
 
 Reference: [Maven Settings > Repositories](http://maven.apache.org/settings.html#Plugin_Repositories)
 
+### `plugin_groups`
+**Optional** json array of plugin groups to add to settings.xml
+
+Reference: [Maven Settings > Profiles](http://maven.apache.org/settings.html#Plugin_Groups)
```
Comment (zteater) [inline README.md:51]: Change the link text to `Maven Settings > Plugin Groups`

**Review by zteater — CHANGES_REQUESTED** [review]
This looks really good, thank you for adding this feature. Just 1 minor update to the readme.

**Review by zteater — COMMENTED** [review]
Also, please change the target branch to `whelk-io:develop`

File: README.md (hunk 1 above)
Comment (uportalis) [reply README.md]: Fixed. Sorry, bad copy-paste :-/

**Review by zteater — APPROVED** [review]
Looks good!

**Comment by zteater** [comment]
@uportalis this has been added to v14

**Pull request merged**